auth/auth/templates/users.html

```diff
@@ -1,53 +1,76 @@
+{% from "utils.html" import submit_button, danger_button %}
 {% extends "layout.html" %}
 {% block title %}Users{% endblock %}
 {% block content %}
-  <h1>Create User</h1>
-  <form action="{{ base_path }}/users" method="POST">
-    <div>Username: <input required name="username" /></div>
-    <div>Login ID: <input name="login_id" /></div>
-    <div><input type="checkbox" name="is_developer" value="1" /> Developer</div>
-    <div><input type="checkbox" name="is_service_account" value="1" /> Service Account</div>
-    <input type="hidden" name="_csrf" value="{{ csrf_token }}"/>
-    <button>
-      Create
-    </button>
-  </form>
+<div class='w-full md:w-2/5 flex flex-col md:flex-row justify-between'>
+  <div>
+    <h1 class='text-2xl font-light mb-4'>Create User</h1>
+    <form action="{{ base_path }}/users" method="POST">
+      <div class='flex-col space-y-2'>
+        <div class='flex justify-between space-x-4'>
+          <span>Username</span>
+          <input class='border rounded-sm p-1 text-sm' required name="username" />
+        </div>
+        <div class='flex justify-between space-x-4'>
+          <span>Login ID</span>
+          <input class='border rounded-sm p-1 text-sm' name="login_id" />
+        </div>
+        <div><input type="checkbox" name="is_developer" value="1" /> Developer</div>
+        <div><input type="checkbox" name="is_service_account" value="1" /> Service Account</div>
+        <input type="hidden" name="_csrf" value="{{ csrf_token }}" />
+        <div class='flex justify-end'>
+          {{ submit_button('Create') }}
+        </div>
+      </div>
+    </form>
+  </div>
+  <div>
+    <h1 class='text-2xl font-light mb-4'>Delete User</h1>
```

Comment:
these two forms are a little close together, would you mind adding a left margin to the delete user one?

Reply:
Added some spacing between the two elements in the flexbox